src/Core/Framework/Gateway/Context/Command/Handler/AbstractContextGatewayCommandHandler.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Gateway\Context\Command\Handler;
+
+use Shopware\Core\Framework\Gateway\Context\Command\AbstractContextGatewayCommand;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+
+#[Package('framework')]
+abstract class AbstractContextGatewayCommandHandler
```

Comment:
this class could have a generic type of the handled command, which then could be used on the both methods. then you do not need to overwrite the PHPDoc in every implementation.